docs/business-apps/power-apps/get-started/create-your-first-custom-form.md

```diff
@@ -60,10 +60,14 @@ Before starting, make sure your SharePoint site is set up with the [required lis
 
     ![Set image border radius](../../../images/gs01-set-image-border-radius.png)
 
-1. To change the image of **Image1** from the sample image to the owner's profile photo, make sure you select **Image1**. In the formula bar, change the property to **Image**, and enter `SharePointIntegration.Selected.Owner.Picture` as the formula. The user photo from the owner's Microsoft 365 user profile appears in **Image1**.
+1. To change the image of **Image1** from the sample image to the project owner's profile photo, you can use the Office 365 Users connector to retrieve the photo by the owner's email. First, [connect your custom form to the Office 365 Users connector](https://docs.microsoft.com/powerapps/maker/canvas-apps/connections/connection-office365-users).
```

Comment:
'Office 365' or the rebranded 'Microsoft 365' Users connector?

Reply:
The connectors in the UI shows up as "Office 365 Users" so I think the tutorial wording should match to avoid confusion. Is there another wording change here that you want me to make?

Reply:
I'll update the other step to say "Office 365"